packages/flutter/test/cupertino/page_test.dart

```diff
@@ -285,15 +291,15 @@ void main() {
     );
 
     await tester.pump();
-    await tester.pump(const Duration(milliseconds: 400));
+    await tester.pumpAndSettle();
```

Comment:
In general, we should check that all animations complete in some reasonable amount of time.